scripts/testBot/runAqaArgParse.py

```diff
@@ -49,7 +54,7 @@ def main():
     parser.add_argument('--sdk_resource', default=['nightly'], choices=['nightly', 'releases', 'customized'], nargs='+')
     parser.add_argument('--customized_sdk_url', default=['None'], nargs='+')
     parser.add_argument('--archive_extension', default=['.tar'], choices=['.zip', '.tar', '.7z'], nargs='+')
-    parser.add_argument('--build_list', default=['openjdk'], choices=['openjdk', 'functional', 'system', 'perf', 'external'], nargs='+')
+    parser.add_argument('--build_list', default=['openjdk'], choices=['openjdk', 'functional', 'system', 'perf', allFiles], nargs='+')
```

Comment:
@sophia-guo - is allFiles the correct replacement here?

Reply:
No, I believe we are expecting comma separated strings instead of a list.